bin/build.sh

```diff
@@ -15,17 +15,16 @@
 # - Z3
 # - Boogie
 # - Corral
-# - Symbooglix
```

Comment:
Don't remove Symbooglix. I think you are not looking at the latest develop maybe.